### ISSUE TITLE
Is there a reference for .travis.yml file?

### ISSUE THREAD

**Issue by debuggy** [travis-ci/docs-travis-ci-com#2087]
Hi there, I wonder if there is a reference doc or schema for the .travis.yml file. So I could figure out the travis configuration globally, thanks!

**Comment by rutujar** [comment]
Yes, You can refer to [TRAVIS CI](https://docs.travis-ci.com/user/languages/) or Refer My repository which is implemented for 8 differenet languages (https://github.com/rutujar/helloworld_travis_demo)

**Comment by AlphaSquadAlgorithms** [comment]
Yes, You can refer to TRAVIS CI or Refer My repository which is implemented for 8 differenet languages (https://github.com/AlphSquadAlgorithms/helloworld_travis_demo)

**Comment by yapcwed** [comment]
There's the [JSON Schema](http://json.schemastore.org/travis) that is used for autocompletion/ validation by various [editors](http://schemastore.org/json#editors). E.g. VSCode with the [YAML Package](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml).

**Comment by yapcwed** [comment]
Duplicate of #2004
